R/latest-dependencies.R

```diff
@@ -0,0 +1,39 @@
+#' Use "latest" versions of all dependencies
+#'
+#' Pins minimum version of dependencies to latest (as determined by "source)"
+#' Useful for tidyverse package, but should otherwise be used with extreme
+#' care.
+#'
+#' @keywords internal
+#' @export
+#' @param overwrite By default (`FALSE`), only dependencies without version
+#'   specifications will be modified. Set to `TRUE` to modify all dependencies.
+#' @param source Use "local" or "CRAN" package versions.
+use_latest_dependencies <- function(overwrite = FALSE, source = c("local", "CRAN")) {
```

Comment:
I would have done CRAN then local. Do we really want local to be the default?

Reply:
I don't think it matters (especially since I only accidentally included in this PR)